tests/components/device_tracker/test_owntracks.py

```diff
@@ -650,3 +661,14 @@ def test_waypoint_import_existing(self):
         self.send_message(WAYPOINT_TOPIC, waypoints_message)
         new_wayp = self.hass.states.get(WAYPOINT_ENTITY_NAMES[0])
         self.assertTrue(wayp == new_wayp)
+
+    def test_encrypted_payload(self):
```

Comment:
Let's skip the test that does the full encryption run by default unless we start tests with `LIBSODIUM=RUN py.test`.

```python
import pytest

@pytest.mark.skipif("os.environ.get('LIBSODIUM') != 'RUN'")
def test_encrypted_payload(self):
    …
```

Still would need tests with the libnacl stuff mocked out to make sure the other logic is solid (passing keys based on topic etc)